src/Nethermind/Nethermind.Evm/Tracing/ParityStyle/ParityAction.cs

```diff
@@ -28,4 +33,198 @@ public class ParityTraceAction
+                {
+                    reader.Read();
+                    value.Result = JsonSerializer.Deserialize<ParityTraceResult?>(ref reader, options);
+                }
+                else if (reader.ValueTextEquals("subtraces"u8))
+                {
+                    reader.Read();
+                    value.Subtraces = JsonSerializer.Deserialize<List<ParityTraceAction>>(ref reader, options);
+                }
+                else if (reader.ValueTextEquals("author"u8))
+                {
+                    reader.Read();
+                    value.Author = JsonSerializer.Deserialize<Address?>(ref reader, options);
+                }
+                else if (reader.ValueTextEquals("rewardType"u8))
+                {
+                    reader.Read();
+                    value.RewardType = JsonSerializer.Deserialize<string?>(ref reader, options);
+                }
+                else if (reader.ValueTextEquals("error"u8))
+                {
+                    reader.Read();
+                    value.Error = JsonSerializer.Deserialize<string?>(ref reader, options);
+                }
+                else if (reader.ValueTextEquals("traceAddress"u8))
+                {
+                    reader.Read();
+                    value.TraceAddress = JsonSerializer.Deserialize<int[]?>(ref reader, options);
+                }
+                else if (reader.ValueTextEquals("includeInTrace"u8))
+                {
+                    reader.Read();
+                    value.IncludeInTrace = reader.GetBoolean();
+                }
+                else if (reader.ValueTextEquals("isPrecompiled"u8))
+                {
+                    reader.Read();
+                    value.IsPrecompiled = reader.GetBoolean();
+                }
+
+                reader.Read();
+            }
+
+            return value;
+        }
+
+        public override void Write(
+            Utf8JsonWriter writer,
+            ParityTraceAction value,
+            JsonSerializerOptions options)
+        {
+            if (value.Type == "reward")
+            {
+                WriteRewardJson(writer, value, options);
+                return;
+            }
+
+            if (value.Type == "suicide")
+            {
+                WriteSelfDestructJson(writer, value, options);
+                return;
+            }
+            writer.WriteStartObject();
+
+            if (value.CallType != "create")
+            {
+                writer.WriteString("callType"u8, value.CallType);
+            }
+            else
+            {
+                writer.WriteString("creationMethod"u8, value.CreationMethod);
+            }
+
+            writer.WritePropertyName("from"u8);
+            JsonSerializer.Serialize(writer, value.From, options);
+            writer.WritePropertyName("gas"u8);
+            JsonSerializer.Serialize(writer, value.Gas, options);
+
+            writer.WritePropertyName("input"u8);
```

Comment:
In the original converted, there is a `init` property for `create`. Why removed?
```
if (value.CallType == "create")
            {
                writer.WriteProperty("init", value.Input, serializer);
            }
            else
            {
                writer.WriteProperty("input", value.Input, serializer);
                writer.WriteProperty("to", value.To, serializer);
            }
```

Reply:
I couldn't find it in any docs or output tracer they are all expecting `input`